browser/tor/tor_profile_service_factory.cc

```diff
@@ -38,14 +39,14 @@ TorProfileServiceFactory* TorProfileServiceFactory::GetInstance() {
 // static
 void TorProfileServiceFactory::SetTorDisabled(bool disabled) {
   if (g_brave_browser_process)
-    g_brave_browser_process->local_state()->SetBoolean(tor::prefs::kTorDisabled,
-                                                       disabled);
+    g_browser_process->local_state()->SetBoolean(tor::prefs::kTorDisabled,
+                                                 disabled);
 }
 
 // static
 bool TorProfileServiceFactory::IsTorDisabled() {
   if (g_brave_browser_process)
```

Comment:
`g_browser_process`?

Reply:
yes, `local_state` is part of g_browser_process

Reply:
oh, sorry, was looking at the wrong thing

Reply:
seems a bit odd though, when would this ever be null @darkdh ?

Reply:
maybe you have a test that should be using testing_browser_process ?

Reply:
it should never be null